### ISSUE TITLE
Regarding test dataset 

### ISSUE THREAD

**Issue by PartheshSoni** [Zhongdao/Towards-Realtime-MOT#84]
What should be the format of test data? Is it exactly same as training data. Also, what information is required in text files?

**Comment by Zhongdao** [comment]
The format of test data is the same as [MOT challenge](https://motchallenge.net/).
If you need to evaluate tracking accuracy in terms of CLEAR metric (e.g., MOTA), the ground-truth annotation is required, please see the format of MOT15/16/17.

**Comment by PartheshSoni** [comment]
Thanks a lot! You are awesome!!
